lisa/tools/dmesg.py

```diff
@@ -81,6 +87,27 @@ def get_vmbus_version(self) -> VersionInfo:
                 return VersionInfo(int(major), int(minor))
         raise LisaException("No find matched vmbus version in dmesg")
 
+    def get_isolation_config(self) -> dict[str, str]:
```

Comment:
It's CVM only logic, not used by other test cases. So it doesn't need to add on dmesg level.

Reply:
Alright will move this logic within cvm.py